src/main/java/com/dev/cinema/Main.java

```diff
@@ -64,12 +69,16 @@ public static void main(String[] args) throws AuthenticationException {
                 (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
         User user1 = userService.findByEmail("mixmix@gmail.com").get();
         shoppingCartService.addSession(movieSession, user1);
-        System.out.println(shoppingCartService.getByUser(user1));
+        try {
+            shoppingCartService.getByUser(user1);
+        } catch (Exception e) {
```

Comment:
which exception can be thrown by this method?

Reply:
done